Packs/IBMResilientSystems/Integrations/IBMResilientSystems/README.md

```diff
@@ -205,6 +207,10 @@ Severity:Low</pre>
 <td style="width: 211px;">nist</td>
 <td style="width: 503px;">NIST Attack Vectors (added to the current list of NIST attack vendors)</td>
 </tr>
+<tr>
+<td style="width: 211px;">other-fields</td>
+<td style="width: 503px;">A json object of the form: {field_name: new_field_value} currently we support the following field types<br /><img src="https://github.com/demisto/content/raw/3322c5933388f2ea9c52dc9fe31a5feb52bc1050/Packs/IBMResilientSystems/doc_files/support_field_types.png" /></td>
```

Comment:
This description is not the same as the one from the command. Please add it here, and keep the image you added afterwards.